server.js

```diff
@@ -1,21 +1,61 @@
+import dotenv from 'dotenv'
 import express from 'express'
-import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
+import members from './data/technigo-members.json'
+import roles from './data/technigo-roles.json'
+import { userName, password } from './keys'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+dotenv.config()
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/mongoApi"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
+const memberSchema = new mongoose.Schema({
+  name: String,
+  surname: String,
+  role: {
+    type: mongoose.Schema.Types.ObjectId,
```

Comment:
This is a bit next level from what I did and I really like the detail of the role as a refernce to another schema: seems useful and probably common in real life to keep objects neet and DBs in check :)